src/storage/btree.c

```diff
@@ -22284,7 +22310,7 @@ btree_scan_for_show_index_capacity (THREAD_ENTRY * thread_p, DB_VALUE ** out_val
   db_make_int (out_values[idx], cpc.dis_key_cnt);
   idx++;
 
-  db_make_int (out_values[idx], cpc.tot_val_cnt);
+  db_make_bigint (out_values[idx], cpc.tot_val_cnt);
```

Comment:
How about adding a comment for the column name above each out_values?

Reply:
OK, I will add comments.